.github/workflows/tests.yml

```diff
@@ -46,6 +48,17 @@ jobs:
       - name: Code-style
         run: cargo fmt -- --check
 
+      - name: get Dove
+        run: |
+          curl -sL --fail \
+            -H "Accept: application/octet-stream" \
+            -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+            -o "dove" \
+            "https://github.com/pontem-network/move-tools/releases/download/1.2.1/dove-1.2.1-${{ matrix.cfg.dove }}"
+          sudo chown runner ./dove && chmod +x ./dove
+          mv ./dove ~/.cargo/bin/dove
+          dove -V
```

Comment:
Use [dove action](https://github.com/marketplace/actions/get-dove) instead of curl, please.
```yaml
      - name: Fetch latest move-tools dove binary
        uses: pontem-network/get-dove@main
        with:
          version: ${{ matrix.dove-version }}
          token: ${{ secrets.GITHUB_TOKEN }}
```